tests/unit/test_cli_flags.py

```diff
@@ -45,25 +45,54 @@ def test_log_path_default(self, run_context):
         assert getattr(flags, "LOG_PATH") == "logs"
 
     @pytest.mark.parametrize(
-        "do_not_track,expected_anonymous_usage_stats",
+        "set_stats_param,do_not_track,expected_anonymous_usage_stats",
```

Comment:
@MichelleArk 
Hilariously the tests were still passing because they were just testing that DO_NOT_TRACK was honored but since boolean logic is easy to get wrong, especially when renaming things I expanded the test matrix a bit. :)

Reply:
Thank you!! Looks great.